rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -43,7 +43,7 @@ Why should we *not* do this?
 [alternatives]: #alternatives
 
 - It's time to establish a clear, reliable process for proposing changes and tracking the decisions we've made. If done right, this will do a lot to reduce confusion, encourage participation, and encourage a high level of transparency around important decisions that impact the project.
-- Rather than starting from scratch, we're starting with the process that is being used, successfully, by a project we admire -- the Rust language
+- Rather than starting from scratch, we're starting with the process that is being used, successfully, by a project we admire -- the IPFS language
```

Comment:
reference to Rust was removed from here, I think the regex was a bit overzealous.

Reply:
thanks. fixed it.